### DISCUSSION TITLE
What exactly are the risks for clicking the "Load missing entries" button in the friends list tab?

### DISCUSSION THREAD

**Discussion by Rexios80** [vrcx-team/VRCX#175] [Q&A] [answered]
Is calling the API that many times to get the data possibly going to cause a ban?

**Comment by Myrkie** [comment]
depends, right now VRCX has its own user agent for api requests by request of a vrchat team member

**Comment by Natsumi-sama** [comment, marked as answer]
I'm not sure hence the warning use at your own risk.
